### PR TITLE
🐛 Fixed video in README

### PR DESCRIPTION
**Pull request by chedabob**
### Description

The video in the README was just showing as the broken file icon on Firefox and Chrome.

image

### Checklist
- [ ] I've added at least one test that validates that my change is working, if appropriate
- [ ] I've followed the code style of the rest of the project
- [x] I've read the [Contribution Guidelines](CONTRIBUTING.md)
- [x] I've updated the documentation if necessary

❤️ Thanks for contributing to the 🏈 Rugby!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,9 +2,9 @@
 <p align="center">
   <img src="https://user-images.githubusercontent.com/64660122/140398205-9328806e-897d-483d-a898-c90f66840196.png" width="600"/>
 </p>
-<p align="center">
-  <img src="https://user-images.githubusercontent.com/64660122/141674975-22c970d6-943e-49cc-a86a-42276b3520e8.mp4" width="600"/>
```
Comment (swiftyfinch) [inline README.md:6]: Hello! Thank you for your pull request. I think the better way to change this video to gif.
I already uploaded one. Could you just replace the link?
```
https://user-images.githubusercontent.com/64660122/194708589-7331a02a-6d6e-4c0f-a7ec-e367f7228080.gif
```

**Review by swiftyfinch — CHANGES_REQUESTED** [review]

**Comment by swiftyfinch** [comment]
Also, feel free to leave empty checkmarks in todo list. Sometimes it's OK.

**Comment by swiftyfinch** [comment]
Also, I changed pull request title. It's better to write "What you did in such request", because it will use in the new version changelog.

File: README.md (hunk 1 above)
Comment (chedabob) [reply README.md:6]: Ah wasn't sure if it was intentional it didn't auto-play. Have updated it to use the Gif.

File: README.md [hunk 2]
```diff
@@ -2,9 +2,9 @@
...
-</p>
+
+![Rugby Demo](https://user-images.githubusercontent.com/64660122/194708589-7331a02a-6d6e-4c0f-a7ec-e367f7228080.gif)
```
Comment (swiftyfinch) [inline README.md:6]: Why we need to change HTML layout and image width here? I don't see any reason for this.
Please, change only the link. All other lines should be kept unchanged.
```suggestion
<p align="center">
   <img src="https://user-images.githubusercontent.com/64660122/194708589-7331a02a-6d6e-4c0f-a7ec-e367f7228080.gif" width="600"/>
</p>
```

**Review by swiftyfinch — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (swiftyfinch) [reply README.md:6]: Yeah, the goal is enable auto-play as it was working in Safari.

File: README.md (hunk 2 above)
Comment (swiftyfinch) [reply README.md:6]: I made similar changes in my another repository. You can check it out here:
https://github.com/swiftyfinch/Pods/commit/c35db89c480ecfb6d0187cbc78deec8ba8905cf2

**Review by swiftyfinch — APPROVED** [review]

**Comment by swiftyfinch** [comment]
@chedabob Hey! I see that you don't included in list of 🏈 Rugby contributors.
I think it can be so because you used different (not GitHub) email while committing.
But I see that count of contributors changed. It's pretty strange.

You can read more here or ask GitHub support.
https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/why-are-my-contributions-not-showing-up-on-my-profile

**Pull request merged**